### PR TITLE
feat: alert and button

### PR DESCRIPTION
**Pull request by amirnathani9**
## Change Logs
# Screenshots
alert

# Preview Link
https://deploy-preview-2--nomark-ui.netlify.app/

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.primary-btn{
+    background-color: var(--primary-color);
+    color: var(--text-color-1);
+}
+
+.secondary-btn{
+    background-color: var(--secondary-color);
+    color: var(--text-color-2);
+}
```
Comment (Ashishgupta08) [inline components/button/button.css:15]: create a class `btn` and put styles of `utilities` classes

File: components/alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                <li><a href="../avatar/avatar.html" class="sidebar-items">Avatar</a></li>
+                <li><a href="./alert.html" class="sidebar-items active">Alert</a></li>
+                <li><a href="../badge/badge.html" class="sidebar-items">Badge</a></li>
+                <li><a href="" class="sidebar-items">Button</a></li>
+                <li><a href="" class="sidebar-items">Card</a></li>
+                <li><a href="" class="sidebar-items">Modal</a></li>
+            </ul>
+        </div>
+        <!-- Alert -->
+        <div class="click-remove">
```
Comment (Ashishgupta08) [inline components/alert/alert.html:48]: write better class names, `class` name should be SELF EXPLANATORY

**Review by Ashishgupta08 — CHANGES_REQUESTED** [review]
Ui is really good.
Fix the navbar on the left

Make Components-based classes as well through which users can directly create components without providing utility classes and provide flexibility to the user through the `utility` classes.

**Review by xii-vi — APPROVED** [review]
_Work on the suggested comments. After that, you can merge._

File: components/button/button.css (hunk 1 above)
Comment (amirnathani9) [reply components/button/button.css:15]: done with the changes

**Pull request merged**